frank/make_figs.py

```diff
@@ -262,6 +262,11 @@ def make_quick_fig(u, v, vis, weights, sol, bin_widths, dist=None,
 
         plot_brightness_profile(sol.r, sol.I / 1e10, ax1, c='r', label='frank')
 
+        if hasattr(sol, '_nonneg'): 
+            plot_brightness_profile(sol.r, sol._nonneg / 1e10, ax0, ls='--', c='#009933', label='non-neg.')
+            plot_brightness_profile(sol.r, sol._nonneg / 1e10, ax1, ls='--', c='#009933', label='non-neg.')
+
+
```

Comment:
I know it's from old code, but it would be better to have not have `nonneg` private (`_nonneg`) because you want people to access it.

Reply:
Good point, thanks! Changed to non-hidden in [22907d7](https://github.com/discsim/frank/pull/200/commits/22907d79ec3128f51103ab76a8d949d632ab2ac9)